Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function addVAT(price) {
+  const vatAdded = price * 1.2;
+  return `£${vatAdded}`;
```

Comment:
It looks like the price parameter should always be in pounds. How could you rename the parameter name `price` on line 12 to make that clearer to someone who will use your function?

Reply:
priceInPounds?